msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java

```diff
@@ -1371,23 +1382,24 @@ public String processPvtMsgReply() {
    * @return - pvtMsgForward
    */ 
   public String processPvtMsgForward() {
-	    log.debug("processPvtMsgForward()");
```

Comment:
Could you undo all the indentation changes on the existing lines please, let's just adjust the indentation of  the code added.